src/api/app/queries/outdated_notifications_finder/bs_request.rb

```diff
@@ -0,0 +1,10 @@
+class OutdatedNotificationsFinder::BsRequest
+  def initialize(scope, notifiable)
+    @scope = scope
+    @notifiable = notifiable
+  end
+
+  def call
+    @scope.where(notifiable_type: 'BsRequest', notifiable_id: @notifiable.id)
```

Comment:
All of this code, just for a simple `.where`?

Reply:
At first this was going to be something that covered several models, and the idea was to start with the ones we knew and plug in the rest, but in the end we have two. I'll try to make it simpler